File: C Codes/conditional Statement/nestedif.c
#include<stdio.h>

int main()
{
    int a,b,c;
    printf("Enter 3 number:");
    scanf("%d %d %d", &a, &b, &c);

    if(a>b)  //100>200
    {
        if(a>c) { 
            printf("%d is Greater", a);
        }
        else {
            printf("%d is Greater", c);
        }
    }
    else if(b>c) { //200>300
        printf("%d is Greater", b);
    }
    else {
        printf("%d is Greater", c);
    }

}

File: Python code/Advance/nestedif.py
num = int(input("Enter a number")) #15

if(num%3 == 0): #15 % 3= 0==0 true
    if(num%5 == 0): # 15 % 5 =0 == 0 true
        print("Number is divisible by 3 & 5")
    else:
        print("Number is divisible by 3 but not 5")
elif(num%5 ==0 ):
    print("Number is not divisible by 3 but divisible by 5")
else:
    print("Number is not divisible by 3 & 5")
    